### ISSUE TITLE
Launchpad build window opens outside screen

### ISSUE THREAD

**Issue by DungeonTroll** [taniwha/Extraplanetary-Launchpads#196]
The launchpad build window will open in the corner of my screen, outside my click range.
is there any way to fix this?

![quite problematic](https://user-images.githubusercontent.com/113061439/188966208-448694ba-c005-4b3b-bc92-caaf127ba734.PNG)

**Comment by taniwha** [comment]
Yes, there is a way: edit GameData/ExtraplanetaryLaunchpads/Plugins/PluginData/Launchpad/Settings.cfg and adjust the position for MainWindow.

**Comment by DungeonTroll** [comment]
> Yes, there is a way: edit GameData/ExtraplanetaryLaunchpads/Plugins/PluginData/Launchpad/Settings.cfg and adjust the position for MainWindow.

thank you, it worked like a charm!
